core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionEvaluatorTest.scala

```diff
@@ -418,7 +418,7 @@ class ExpressionEvaluatorTest extends AnyFunSuite with TestDriverComponent {
       }
 
       assertResult(Map("sample1" -> TrySuccess(Seq(AttributeString("sample1"))))) {
-        runAndWait(evalFinalAttribute(workspaceContext, "Sample", "sample1", "this.sample_id"))
+        runAndWait(evalFinalAttribute(workspaceContext, "Sample", "sample1", "this.Sample_id"))
```

Comment:
this pre-existing test started to fail with the new db collation. It used an entity type of "Sample" (sentence case) but an expression of "this.sample_id" (lower case). With case-insensitive matching, this passed, but now fails. I've updated the test to be accurate, but this behavior change will need to be publicized.